app/controllers/surveys_controller.rb

```diff
@@ -1,4 +1,35 @@
 class SurveysController < ApplicationController
+
   def send_survey
```

Comment:
nit: maybe just `send` given that the route already lives on the /survey resource?